examples/shared-worker/package.json

```diff
@@ -63,12 +63,9 @@
     "dependencies": {
         "@microsoft/applicationinsights-shims": "3.0.1",
         "@microsoft/dynamicproto-js": "^2.0.3",
-        "@microsoft/applicationinsights-web": "3.0.8",
-        "@microsoft/applicationinsights-web-snippet": "1.1.6",
-        "@microsoft/applicationinsights-core-js": "3.0.8",
+        "@microsoft/applicationinsights-web": "3.1.0",
+        "@microsoft/applicationinsights-web-snippet": "1.1.0",
```

Comment:
The last released version of this package was 1.0.1, so rolling back this number to the next release